salt/elastic-fleet-package-registry/soc_elastic-fleet-package-registry.yaml

```diff
@@ -1,4 +1,4 @@
 elastic_fleet_package_registry:
   enabled:
-    description: You can enable or disable Elastic Fleet Package Registry.
+    description: Enables or disables the Fleet package registry process. This process must remain enabled for proper HIDS functionality.
```

Comment:
Enables or disables the Fleet package registry process. This process must remain enabled to allow Elastic Agent packages to be updated.